src/Avalonia.Diagnostics/Diagnostics/DevTools.cs

```diff
@@ -38,15 +46,21 @@ public static IDisposable Open(TopLevel root)
             {
                 window = new MainWindow
                 {
-                    Width = 1024,
-                    Height = 512,
+                    Width = options.Size.Width,
+                    Height = options.Size.Height,
                     Root = root,
                 };
 
                 window.Closed += DevToolsClosed;
+                window.ViewModel.ShouldVisualizeDirtyRects = options.ShouldVisualizeDirtyRects;
```

Comment:
This can already be controlled by using `window.Renderer.VisualizeDirtyRects` - I don't think it's necessary here; DevTools just provides a UI to toggle this property, if you want to control it programmatically you should use the `Renderer` property.